src/components/common/Search.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <form class="search-form"
```

Comment:
I really like the use of the common search bar. It might look good with a border that brings the input and button together. There are some classes in index.css that give the border to the button and input '.card-search__input', '.card-search__input-btn'. Widget 3 is using them but feel free to change the style to whatever you feel is best. I tried them on your widget with a margin-top on the map to give a bit more vertical space and turned out like the screenshot attached. I'm not married to the border colour or being full-width though - what do you think?
<img width="432" alt="Screenshot 2021-05-21 at 16 32 17" src="https://user-images.githubusercontent.com/18268215/119163141-07bfe300-ba53-11eb-965c-939fd8fc492d.png">

Reply:
Ooh love this -- have added your styles to my widget, and also have the map a border-radius to match the widget box.

I do actually like the idea of full width. Then if the user adds a full address, they can see it all.

Reply:
Good point about the long addresses. Looks great!